### DISCUSSION TITLE
App is down

### DISCUSSION THREAD

**Discussion by jaycollier** [lukevella/rallly#1248] [General]
Seeing this on all app pages:

400: BAD_REQUEST
Code: BAD_REQUEST
ID: iad1::2rt4r-1723059629083-5326ff8e0908

**Comment by jaycollier** [comment]
And now back!

**Reply by lukevella** [reply]
Hi, I can see that some requests have been blocked and the reason given was a "malicious client". Can you provide information about your browse/os?

**Reply by jaycollier** [reply]
Up-to-date Mac Chrome with cleared cache and Duck Duck Go with privacy tools activated. Also Windows browsers at our local community college on a different network. The site is back, but if you would like more info, just let me know! (Love the app!)
